designs/administrator/future-openshift/resource-related-quick-starts/index.md

```diff
@@ -0,0 +1,65 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Surfacing resource-related quick starts
+
+## Introduction
+Sometimes quick starts can be closely associated with a resource type, for example, a virtual machine or a pipeline. With that in mind, it became important to surface quick starts on that resource's page to improve the flow of both creating a resource and using a resource. This design demonstrates how to do so.
+
+## Design summary
+The design is broken up into scenarios:
+- [Populated resource list views](#populated-resource-list-views)
+- [Empty resource list views](#empty-resource-list-views)
+
+Whether the list view is populated or not, the general pattern of this design includes surfacing a dismissible label to make it clear that a quick start is available within the page heading.
+![general-design](img/general-design.png)
+
+The label is more like a hint and can be hidden, which would then hide **all** resource-related quick start labels for that user.
+![general-design-hide](img/general-design-hide.png)
+
+If there are multiple quick starts associated with a resource, they should be shown in a side panel, where the order logic should be as follows:
+- In progress quick starts
+- Not started quick starts
+- Do not show completed quick starts
+![side-panel](img/side-panel.png)
+
+At the end of the list, there should be a link to view all quick starts associated with that resource. The button should take the user to the quick starts catalog page that is filtered by the resource type.
+![side-panel-cta](img/side-panel-cta.png)
+
+## Empty resource list views
+If a quick start exists that will help the user create a resource type, it should be made available within the empty state of the list view.
+![empty-list](img/empty-list.png)
+
```

Comment:
I am a little concerned that if we just use the name of the quick start as the button label, it will not aways be clear to the user. Could we do the same thing we do below for the multiple scenario where we have a label that starts with "Getting started with...". We could away just say "Getting started with " + <resource type> (i.e. "Getting Started with Volume Snapshots").

Reply:
+1

Reply:
Because this is a button label, I recommend that it be action-oriented to align with PatternFly's button guidelines and to give the user more info about what happens when they click. 

In some areas of Managed Kafka, the secondary quick start buttons in empty states read, "Access the quick start guide." With that general language, we also don't have to worry about updating button text if the quick start title ever changes.

Reply:
Also, here is some copy to use in the empty state instead of the placeholder copy: 

"To get started, create a Volume Snapshot."

Reply:
Yes that could definitely work! So I'll update button text to be "Get started with [resource]" for the single quick starts as well